node/genesis.toml

```diff
@@ -0,0 +1,21 @@
+# This is an example configuration file for the Miden node.
```

Comment:
We should update this text - i.e., this is an example genesis input file rather than a configuration file for the ndoe.